domain/tshirt.js

```diff
@@ -10,19 +10,26 @@ function invalidData(body) {
   return _.isUndefined(body.size) || _.isUndefined(body.model);
 }
 
+tshirt.NoTShirt = function () {
+  return {
+    details: [{ size: "L", model: "Slim fit" }],
```

Comment:
In that case, this is a concern of the route and not really the tshirt domain object itself.